src/content/docs/discovery/index.mdx

```diff
@@ -29,12 +31,73 @@ The next step is to plan, build, optimize, and launch a production-ready storefr
 
 </Callouts>
 
+## Tips for a successful project
```

Comment:
I think we should combine the "tips for a successful launch" section with the "[project scope](https://commerce-docs.github.io/microsite-commerce-storefront/discovery/#project-scope)" section. Thoughts?

Reply:
I think this would make sense.